.gitignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
- [ ] Kindly add `test.md` file to `.gitignore` as it is in the project requirements

Reply:
I have fixed it now can you check.

Reply:
@NatanimA Hey, Good job on adding the file to `.gitignore` 👍🏻 Kindly ask for another review from dashboard 😄